pkg/operator/certrotation/target.go

```diff
@@ -102,6 +104,36 @@ func needNewTargetCertKeyPair(annotations map[string]string, caBundleCerts []*x5
 	return true
 }
 
+func needNewTargetCertKeyPairForTime(annotations map[string]string, signer *crypto.CA, validity time.Duration, renewalPercentage float32) bool {
```

Comment:
go doc summing up the general idea would help a lot

Reply:
done